app/components/school_options_form_component.rb

```diff
@@ -0,0 +1,44 @@
+class SchoolOptionsFormComponent < ApplicationComponent
+  attr_reader :model, :url, :search_param, :schools, :hint_link
+
+  def initialize(
+    model:,
+    url:,
+    search_param:,
+    schools:,
+    hint_link:,
+    classes: [],
+    html_attributes: {}
+  )
+    super(classes:, html_attributes:)
+
+    @model = model
+    @url = url
+    @search_param = search_param
+    @schools = schools
+    @hint_link = hint_link
+  end
+
+  def hint_translation
+    if schools.count > School::OPTIONS_PER_PAGE
```

Comment:
I'm wondering if it makes more sense for the `OPTIONS_PER_PAGE` constant to exist on the `SchoolOptionsFormComponent` instead of the `School` model.

The component is the only thing that refers to this value. It also seems to be needed for presentational concerns, which makes me wonder whether it's a bit confusing directly on the `School` model.

What do you think?